StantDrawing/BrushTool/GroupSizeView.swift

```diff
@@ -0,0 +1,287 @@
+        let frame = CGRect(x: 0, y: 0, width: 100, height: 100)
+        
+        self.iconReferenceName   = iconReferenceName
+        self.buttonSizeImageName = buttonSizeImageName
+        self.groupSizeType       = groupSizeType
+        
+        super.init(frame: frame)
+    }
+    
+    func conf(delegate: GroupSizeContract, bundle: Bundle, rootGroupView: CGFloat, heightBaseToCenter: CGFloat, xBaseToCenter: CGFloat) {
+        self.delegate = delegate
+        self.bundle    = bundle
+        groupViewWidth = rootGroupView
+        
+        loadGroupPencilSizeButtons()
+        
+        configureGroupSizeLayout(groupButtons: groupSizeButtons, groupSizeView: self, iconReferenceName: iconReferenceName, heightBaseToCenter: heightBaseToCenter, xBaseToCenter: xBaseToCenter)
+    }
+    
+    func setIdentifierForView(view: UIView, identifierName: String) {
+        view.accessibilityIdentifier = identifierName
+    }
+    
+    private func loadGroupPencilSizeButtons() {
+        let buttonSize1   = Button(imageName: "\(buttonSizeImageName)1", imageColor: blackHexColor, selector: #selector(selectSize))
+        let buttonSize2   = Button(imageName: "\(buttonSizeImageName)2", imageColor: blackHexColor, selector: #selector(selectSize))
+        let buttonSize3   = Button(imageName: "\(buttonSizeImageName)3", imageColor: blackHexColor, selector: #selector(selectSize))
+        let buttonSize4   = Button(imageName: "\(buttonSizeImageName)4", imageColor: blackHexColor, selector: #selector(selectSize))
+        let buttonSize5   = Button(imageName: "\(buttonSizeImageName)5", imageColor: blackHexColor, selector: #selector(selectSize))
+        let buttonSize6   = Button(imageName: "\(buttonSizeImageName)6", imageColor: blackHexColor, selector: #selector(selectSize))
+        
+        buttonSize1.uiButton.tag = 2
+        buttonSize2.uiButton.tag = 5
+        buttonSize3.uiButton.tag = 10
+        buttonSize4.uiButton.tag = 30
+        buttonSize5.uiButton.tag = 55
+        buttonSize6.uiButton.tag = 80
+        
+        buttonSize1.uiButton.frame.size = CGSize(width: 13, height: 13)
+        buttonSize2.uiButton.frame.size = CGSize(width: 18, height: 18)
+        buttonSize3.uiButton.frame.size = CGSize(width: 23, height: 23)
+        buttonSize4.uiButton.frame.size = CGSize(width: 28, height: 28)
+        buttonSize5.uiButton.frame.size = CGSize(width: 33, height: 33)
+        buttonSize6.uiButton.frame.size = CGSize(width: 37, height: 37)
+        
+        groupSizeButtons.append(buttonSize1)
+        groupSizeButtons.append(buttonSize2)
+        groupSizeButtons.append(buttonSize3)
+        groupSizeButtons.append(buttonSize4)
+        groupSizeButtons.append(buttonSize5)
+        groupSizeButtons.append(buttonSize6)
+    }
+    
+    private func configureGroupSizeLayout(groupButtons: [Button], groupSizeView: UIView, iconReferenceName: String, heightBaseToCenter: CGFloat, xBaseToCenter: CGFloat) {
```

Comment:
Thank you for adding the comments above the lines, helped me to understand it :)